src/shared/AchievementDetails.js

```diff
@@ -0,0 +1,67 @@
+import { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './AchievementDetails.css';
+
+import AchievementBadge from './AchievementBadge.js';
+
+class AchievementDetails extends Component {
+  render() {
+    const { id, achievement } = this.props;
+    const { description, completed, img } = achievement;
+    const className = completed ? 'active' : 'inactive';
+
+    // TODO: [layout] Define title, dateEarned, currentValue and maxValue
+    const title = 'Badge title';
+    const dateEarned = 'Earned 9 Aug, 2018 - 7:22 PM';
```

Comment:
Why is this hard-coded?

Reply:
It's not being used yet.
There's a CSS layout commented for that but we need to have a unique title for each badge and store/retrieve the dateEarned in the database.